subsys/net/l2/openthread/openthread_utils.c

```diff
@@ -11,6 +11,10 @@ LOG_MODULE_DECLARE(net_l2_openthread, CONFIG_OPENTHREAD_L2_LOG_LEVEL);
 #include <zephyr/net/net_pkt.h>
 #include <zephyr/net/openthread.h>
 
+#if defined(CONFIG_OPENTHREAD_ZEPHYR_BORDER_ROUTER)
```

Comment:
The diff in this file needs some cleanup, it's hard to tell what actually changed here (did anything actually beside this include?).